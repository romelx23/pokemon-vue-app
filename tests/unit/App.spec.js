import { mount } from "@vue/test-utils";
import { describe } from "vitest";
import App from "@/../../src/App.vue";

describe("App", () => {
    test("should render", () => {
        // validar si hace match con el snapshot
        const wrapper = mount(App);
        expect(wrapper.html()).toMatchSnapshot();
    });
    }
);