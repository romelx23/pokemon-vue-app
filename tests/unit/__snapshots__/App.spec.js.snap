// Vitest Snapshot v1

exports[`App > should render 1`] = `
"<div class=\\"container-pokemon\\" data-v-2108564c=\\"\\" data-v-7a7a37b1=\\"\\">
  <h1 data-v-2108564c=\\"\\">Â¿Quien es este Pokemon?</h1>
  <div class=\\"cargando\\" data-v-2108564c=\\"\\"> Cargando el pokemon... </div>
  <!--v-if-->
</div>"
`;
